app/controllers/api/v1/applications_controller.rb

```diff
@@ -12,7 +12,7 @@ class ApplicationsController < ActionController::API
       def index
         crime_apps = CrimeApplication.all.limit(LIMIT)
 
-        render json: ApplicationSerializer.collection(crime_apps).select(&:valid?)
+        render json: ApplicationSerializer.collection(crime_apps, include_details: true).select(&:valid?)
```

Comment:
I guess going forward it might be good to just remove the need to pass the boolean, as we are returning all the data. But not really a big deal and can be put into another PR etc.